### ISSUE TITLE
[Feature request] Option to display master password during entry

### ISSUE THREAD

**Issue by dfberger** [keeweb/keeweb#455]
It would be useful to have the option to display the master password (passphrase) during entry for long complex passwords.

**Comment by humbleice** [comment]
I agree this would be useful.  I usually need to type my master password a few times before getting it right because of some typo that would be easy to catch if I could see the clear text.

**Comment by cGuille** [comment]
Hello,

Would this feature be welcome? Could I help this happen?

Thanks for making this app!

**Comment by antelle** [comment]
It would be very welcome if it’s not implemented by setting input type=text. In other words, if there’s a way to show the password without having it as a string in memory - it’s ok. But it’s somewhat complicated to implement.

**Comment by cGuille** [comment]
I am not sure I understand. Isn't the string already in memory? It is currently in a `<input type="password" />` element, meaning that it is not displayed, but I assume it must be in memory otherwise you could not use it at all.

Do I miss something?

**Comment by antelle** [comment]
It’s not, you can inspect the value of this input, it’s garbage. The implementation is in secure-input.js. But because of type=password it looks visually the same (dots)

**Comment by cGuille** [comment]
Oh alright, so this is more difficult to do that what I anticipated. I will look into it despite not being very optimistic.

Thanks for your answers!
